src/components/DeckPreview.js

```diff
@@ -8,17 +8,19 @@ class DeckPreview extends React.Component {
   }
 
   render() {
+    const maxLength = 35;
+
     const deck = this.props.deck;
+    if (deck.name.length > maxLength) {
```

Comment:
There is a way to do this with css as well, you prefer manual?

Reply:
Yes - it's ok to line wrap, I just want to limit the actual # of chars